### DISCUSSION TITLE
Language server exits when using socket option

### DISCUSSION THREAD

**Discussion by PhiloNL** [tailwindlabs/tailwindcss#12084] [Help]
**What version of VS Code are you using?**
N/A -> Tailwind CSS Language Server

**What version of Tailwind CSS IntelliSense are you using?**
Tailwind CSS Language Server 0.0.13

**What package manager are you using?**
NPM

**What operating system are you using?**
MacOS

**Describe your issue**

I'm trying to run the language server using sockets and I've followed the installation instructions:

```
tailwindcss-language-server --socket=8888
```

The process just dies instantly without any errors.
Any ideas about what this could be or how I can debug this?

Thanks!

**Comment by bradlc** [comment]
Hey @PhiloNL. My understanding from reading https://github.com/microsoft/vscode-languageserver-node/issues/245 is that you need a client that is listening on the same port that you pass to the server, in your case `8888`. I have tested the language server with the code from the [last comment](https://github.com/microsoft/vscode-languageserver-node/issues/245#issuecomment-336136082) and it seems to work fine. I hope that helps!
